app/services/stripe/invoice_handler.rb

```diff
@@ -0,0 +1,36 @@
+class InvoiceHandler
+  def initialize(event)
+    @invoice = event.data.object
```

Comment:
Is this the invoice object? Can we name it so? And let's only pass in the what's necessary. E.g. if we only need the invoice object, let's only pass in that. Let's also add some yardoc documentation about the type of object that's getting passed into the `initialize` method.